### PR TITLE
Update tasks to account for migration of projects

### PR DESCRIPTION
**Pull request by Sam-Hutchins**
We are updating tasks to account for the migration of projects from pegasus to dashboard in the replication task.  This comes as a required action item from [this PR](https://github.com/code-dot-org/code-dot-org/pull/45480) which moved the two affected tables from pegasus to dashboard.  
## Testing story

@cat5inthecradle tested:

```
> bundle exec rake stack:data:validate RAILS_ENV=production

GetSecretValue: production/cdo/slack_token
GetSecretValue: production/cdo/slack_bot_token
GetSecretValue: production/cdo/db_writer
GetSecretValue: production/cdo/redshift_username
GetSecretValue: production/cdo/redshift_password
Parameters:
RedshiftUsername: REDACTED
RedshiftPassword: REDACTED
Pending update for stack `DATA-production`:
Add AuroraReportingClusterDBParameters [AWS::RDS::DBClusterParameterGroup] 
Modify AuroraReportingReplicaDBParameters [AWS::RDS::DBParameterGroup] Properties (Parameters)
Modify AuroraWriterDBParameters [AWS::RDS::DBParameterGroup] Properties (Parameters)
Modify DMSCronPii [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)
Modify DMSCron [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)
Add PrimaryDBParameters [AWS::RDS::DBParameterGroup] 
Add ReportingDBParameters [AWS::RDS::DBParameterGroup] 

```

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: aws/dms/tasks.yml [hunk 1]
```diff
@@ -23,6 +22,7 @@ cron-pii:
 - dashboard.peer_reviews
 - dashboard.circuit_playground_discount_applications
 - dashboard.circuit_playground_discount_codes
+- dashboard.storage_apps
```
Comment (sureshc) [inline aws/dms/tasks.yml]: `pegasus.storage_apps` was renamed to `dashboard.projects`

File: aws/dms/tasks.yml [hunk 2]
```diff
@@ -76,6 +75,7 @@ cron:
 - dashboard.stages_standards
 - dashboard.standard_categories
 - dashboard.frameworks
+- dashboard.user_storage_ids
```
Comment (sureshc) [inline aws/dms/tasks.yml]: `pegasus.user_storage_ids` was renamed to `dashboard.user_project_storage_ids`

Comment (Sam-Hutchins) [reply aws/dms/tasks.yml]: Thank you for pointing these out; I've updated the code.

**Review by maureensturgeon — APPROVED** [review]

**Review by sureshc — APPROVED** [review]

**Comment by sureshc** [comment]
Drop a note in #infrastructure when you’re ready for us to deploy this change. Let’s wait until the deploy is complete before merging.

**Comment by cat5inthecradle** [comment]
Deployed this change successfully.

```
➜  bundle exec rake stack:data:validate RAILS_ENV=production             
GetSecretValue: production/cdo/slack_token
GetSecretValue: production/cdo/slack_bot_token
GetSecretValue: production/cdo/db_writer
GetSecretValue: production/cdo/redshift_username
GetSecretValue: production/cdo/redshift_password
Parameters:
RedshiftUsername: REDACTED
RedshiftPassword: REDACTED
Pending update for stack `DATA-production`:
Modify DMSCronPii [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)
Modify DMSCron [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)

➜  bundle exec rake stack:data:start RAILS_ENV=production
GetSecretValue: production/cdo/slack_token
GetSecretValue: production/cdo/slack_bot_token
GetSecretValue: production/cdo/db_writer
GetSecretValue: production/cdo/redshift_username
GetSecretValue: production/cdo/redshift_password
Parameters:
RedshiftUsername: REDACTED
RedshiftPassword: REDACTED
Pending update for stack `DATA-production`:
Modify DMSCronPii [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)
Modify DMSCron [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)
Proceed? [y/n]
y
 Stack update requested, waiting for provisioning to complete...
..2022-04-06 22:12:12 UTC- DMSCronPii [UPDATE_IN_PROGRESS]
.2022-04-06 22:12:14 UTC- DMSCron [UPDATE_IN_PROGRESS]
.....2022-04-06 22:12:50 UTC- DMSCronPii [UPDATE_COMPLETE]
..2022-04-06 22:13:03 UTC- DMSCron [UPDATE_COMPLETE]
.2022-04-06 22:13:07 UTC- DATA-production [UPDATE_COMPLETE_CLEANUP_IN_PROGRESS]

Stack update complete.
Outputs:
```

**Pull request merged**